test/index.spec.ts

```diff
@@ -1,12 +1,119 @@
+
+    expect(stack).toHaveResource("AWS::Logs::SubscriptionFilter", {
+      DestinationArn: "forwarder-arn",
+      FilterPattern: "",
+    });
+    expect(stack).toHaveResource("AWS::Logs::SubscriptionFilter", {
+      DestinationArn: "forwarder-arn2",
+      FilterPattern: "",
+    });
+  });
+  it("Subscribes the same forwarder to two different lambda functions via seperate addForwarder function calls",() => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "sa-east-1",
+      },
+    });
+    const nodeLambda = new lambda.Function(stack, "NodeHandler", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromAsset("test"),
+      handler: "hello.handler",
+    });
+    const pythonLambda = new lambda.Function(stack, "PythonHandler", {
+      runtime: lambda.Runtime.PYTHON_3_7,
+      code: lambda.Code.fromAsset("test"),
+      handler: "hello.handler",
+    });
+    addForwarder(stack, [nodeLambda], "forwarder-arn");
+    addForwarder(stack, [pythonLambda], "forwarder-arn");
+    const nodeLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(nodeLambda.functionArn,"forwarder-arn")
+    const pythonLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(pythonLambda.functionArn,"forwarder-arn")
+    expect(nodeLambda.logGroup.node.tryFindChild(nodeLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
+    expect(pythonLambda.logGroup.node.tryFindChild(pythonLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
+  });
+  it("Subscribes two different forwarders to two different lambda functions via seperate addForwarder function calls",() => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "sa-east-1",
+      },
+    });
+    const nodeLambda = new lambda.Function(stack, "NodeHandler", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromAsset("test"),
+      handler: "hello.handler",
+    });
+    const pythonLambda = new lambda.Function(stack, "PythonHandler", {
+      runtime: lambda.Runtime.PYTHON_3_7,
+      code: lambda.Code.fromAsset("test"),
+      handler: "hello.handler",
+    });
+    addForwarder(stack, [nodeLambda], "forwarder-arn");
+    addForwarder(stack, [pythonLambda], "forwarder-arn2");
+    const nodeLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(nodeLambda.functionArn,"forwarder-arn")
+    const pythonLambdaLogGroupSubscriptionFilterName = createSubscriptionFilterName(pythonLambda.functionArn,"forwarder-arn2")
+    expect(nodeLambda.logGroup.node.tryFindChild(nodeLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
+    expect(pythonLambda.logGroup.node.tryFindChild(pythonLambdaLogGroupSubscriptionFilterName)).not.toBeUndefined();
+  });
+  it("Subscribes the same forwarder to two different lambda functions via one addForwarder function call",() => {
```

Comment:
Formatting nit: I would put a blank line between each test, it makes things a bit easier to read.